test/Composer/ComposerLockSpec.hs

```diff
@@ -0,0 +1,75 @@
+{-# language TemplateHaskell #-}
+
+module Composer.ComposerLockSpec
+  ( spec
+  ) where
+
+import Prologue
```

Comment:
`Prologue` no longer exists, and this module isn't using `TemplateHaskell`, so we can remove both of these.
```suggestion
module Composer.ComposerLockSpec
  ( spec
  ) where
```